src/entities/swap/swaps/v2/auraBalSwaps/swapExit.ts

```diff
@@ -0,0 +1,143 @@
+import {
+    decodeAbiParameters,
+    decodeFunctionResult,
+    encodeFunctionData,
+    zeroAddress,
+    Address,
+    Hex,
+} from 'viem';
+import { TokenAmount } from '../../../../tokenAmount';
+import { BALANCER_RELAYER } from '@/utils';
+import { balancerRelayerAbi } from '@/abi';
+import { Relayer } from '@/entities/relayer';
+import { auraBalToken, balWethAddress, auraBAL } from './constants';
+import { getSwapData } from './swap';
+import { AuraBalSwapQueryOutput } from './auraBalSwaps';
```

Comment:
```suggestion
import { AuraBalSwapQueryOutput } from './types';

Reply:
Good catch! Any idea why this doesn't get flagged in CI? Was also the case for joinSwap but I missed both.

Reply:
that is interesting. we would expect the lint check to have been failing before huh

I tried reverting the code for this file and running the `pnpm lint` command which is what the `Lint` job from the `Checks` workflow executes during the GHA. Everything still passed even with broken import :thinking: